packages/selectInput/style.ts

```diff
@@ -39,3 +39,7 @@ export const selectIcon = css`
   top: 50%;
   transform: translateY(-50%);
 `;
+
+export const optionalIcon = css`
+  margin: auto;
```

Comment:
The icon is right up against the text. I recommend changing this to `margin: auto ${spaceXs} auto 0`